suite/testcommon/ocr.go

```diff
@@ -50,83 +50,88 @@ func DeployOCRForEnv(i *OCRSetupInputs, envInit environment.K8sEnvSpecInit) {
 	})
 }
 
-// SetupOCRTest setup for an ocr test
-func SetupOCRTest(i *OCRSetupInputs) {
-	By("Funding nodes and deploying OCR contract", func() {
-		ethAmount, err := i.NetworkInfo.Deployer.CalculateETHForTXs(i.NetworkInfo.Wallets.Default(), i.NetworkInfo.Network.Config(), 2)
-		Expect(err).ShouldNot(HaveOccurred())
-		err = actions.FundChainlinkNodes(
-			i.ChainlinkNodes,
-			i.NetworkInfo.Client,
-			i.DefaultWallet,
-			ethAmount,
-			big.NewFloat(2),
-		)
-		Expect(err).ShouldNot(HaveOccurred())
-
-		// Deploy and config OCR contract
-		deployer, err := contracts.NewContractDeployer(i.NetworkInfo.Client)
-		Expect(err).ShouldNot(HaveOccurred())
+// DeployContracts deploys and funds a certain number of offchain aggregator contracts
+func DeployContracts(i *OCRSetupInputs, nrOfOCRContracts int) {
```

Comment:
Might want to name this something that more explicitly points out it's for OCR only, like `DeployOCRContracts`

Reply:
For sure